### PR TITLE
update dashboard and slo to latest releases

### PR DESCRIPTION
**Pull request by tilleryd**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This PR updates gm-dashboard and gm-slo to their latest versions. The primary change is to support TLS.

2. What **changes to custom.yaml** are required?

n/a

3. Have you documented any additional setup steps or configurations options?

n/a

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

- change `requirements.yaml` to run local charts for dashboard and slo
- deploy with minikube as follows

```
minikube start -p gm-deploy --memory 6144 --cpus 6
helm init
./ci/scripts/install-voyager.sh
helm dep up greymatter
helm install greymatter -f greymatter.yaml -f greymatter-secrets.yaml -f credentials.yaml --set global.environment=kubernetes -n gm-deploy
minikube -p gm-deploy service --https=true voyager-edge
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *.tgz
 custom*.yaml
 credentials.yaml
+template.yaml
```
Comment (chrisbsmith) [inline .gitignore]: Out of curiosity, what are you storing in this file?

Comment (tilleryd) [reply .gitignore]: We sometimes run `helm template ...` to see what the k8s template looks like. We don't want to source control it.

Comment (chrisbsmith) [reply .gitignore]: 👍 thanks

**Comment by tilleryd** [comment]
@kaitmore done, thanks

**Pull request merged**